vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -1037,7 +1037,7 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
         const expectedSectionValues = ['header', 'body', 'footer'];
 
         if(expectedSectionValues.indexOf(eventContext.section) === -1) {
-          throw new Error('Unexpected value for section: ' + eventContext.section);
+          return;
```

Comment:
Quick sanity check: Wouldn't this still throw an exception on the server-side? See here: https://github.com/vaadin/flow-components/blob/master/vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/CellFocusEvent.java#L159-L161

Not sure if that is an issue or not.